observability/tools/readme.md

```diff
@@ -11,3 +11,20 @@ There are a number of modern tools to make systems observable. While identifying
 
 - [Loki](./loki.md)
 - [OpenTelemetry](./OpenTelemetry.md)
+
+## Service Mesh
+
+Leveraging a Service Mesh that follows the [Sidecar Pattern](https://www.oreilly.com/library/view/designing-distributed-systems/9781491983638/ch02.html#:~:text=The%20sidecar%20pattern%20is%20a,first%20is%20the%20application%20container.&text=In%20addition%20to%20the%20application,without%20the%20application%20container's%20knowledge.) quickly sets up a go-to set of metrics, and traces (although traces need to be propagated from incoming requests to outgoing requests manually).
+
+A sidecar works by intercepting all incoming and outgoing traffic to your image. It then adds trace headers to each request and emits a standard set of logs and metrics. These metrics are extremely powerful for observability, allowing every service to leverage a unified set of metrics, including:
+
+All metrics exist for both server side and client side:
```

Comment:
readability: potentially rephrase the line above to include this one, such as "...allowing every service, whether client-side or server-side, to leverage a unified set of metrics including:"

Reply:
done!